src/StaticText.cpp

```diff
@@ -0,0 +1,656 @@
+// Copyright (C) 2002-2012 Nikolaus Gebhardt
+// This file is part of the "Irrlicht Engine".
+// For conditions of distribution and use, see copyright notice in irrlicht.h
```

Comment:
we should copy the copyright from the irrlicht version you took this code as this reference to it might change it's meaning if irrlicht.h copyright notice is changed in future